database/migrations/2019_02_05_083929_add_primary_key_to_submissions.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddPrimaryKeyToSubmissions extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('submissions', function (Blueprint $table) {
+            $table->uuid('id');
+            $table->primary('id');
+        });
+
+        \App\Submission::all()->map(function($submission){
```

Comment:
Please import Submission, and space after `function` and space between closing paren and opening brace please